sdk/ffi/go/README.md

```diff
@@ -0,0 +1,39 @@
+# Go FFI
+> ⚠️ This is an initial version of this library in order to give developers something to experiment with. If you use this code to begin testing out Mixnet integration and run into issues, errors, or have feedback, please feel free to open an issue; feedback from developers trying to use it will help us improve it. If you have questions feel free to reach out via our [Matrix channel](https://matrix.to/#/#dev:nymtech.chat).
+
+This repo contains: 
+* `lib.rs`: an initial version of bindings for interacting with the Mixnet via the Rust SDK from Go. These are essentially match statemtns wrapping imported functions from the `nym-ffi-shared` lib. 
+* `ffi/`: a directory containing: 
+  * the `bindings/` files generated using [`uniffi-bindgen-go`]()
+  * `main.go`: an example of using this library. 
```

Comment:
```suggestion
  * [`main.go`](./ffi/bindings/main.go): an example of using this library. 
```

It was hard to find the example file. A link would make it better?

Reply:
And just wondering if `sdk/ffi/go/ffi` should maybe change to `sdk/ffi/go/example`?

Reply:
Yeah its a bit of an annoying path - I wouldn't change to "example" though, as its the bindings files themselves that are generated.. so maybe `sdk/ffi/go/generated_bindings/`?

Reply:
Did a bit of a rearrange - i think its a bit clearer now